spyder/plugins/editor/utils/editor.py

```diff
@@ -799,3 +802,41 @@ def get_file_language(filename, text=None):
             else:
                 break
     return language
+
+
+def get_default_file_content(template_path, text=None):
+    """Get default file content and enconding parsing a template/using given text."""
```

Comment:
```suggestion
    """
    Get default file content and enconding parsing a template/using given text.
    """
```

To make this docstring fit in 79 columns.